constants.js

```diff
@@ -0,0 +1,10 @@
+// number values indicate milliseconds
```

Comment:
Added file to simplify all files utilizing animation timings as well as setTimeout timings to make code base more readable.